### PR TITLE
Finish-Mobile

### PR DESCRIPTION
**Pull request by Joaquin123455**
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
    - Linter should be set up correctly for this repository.
    - Linter result should be green.
    - There should be no custom changes to linter config.
- Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
    - README file should be descriptive.
    - Commit messages should be meaningful.
    - Pull Request should have a descriptive title telling you what changes are inside.
    - Pull Request should have a short summary describing introduced changes in more detail.
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
    - Branch should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
    - Pull Request is made from 'feature branch' into the `main` branch.

### HTML/CSS requirements

- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -100,3 +100,315 @@ header {
...
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+}
+
+.tags {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] As per the figma design, tags should be left aligned not centered. Also, there is more space between the tags Kindly fix this.

- Figma Design:
- image

- Your Design:
- image

File: style.css [hunk 2]
```diff
@@ -100,3 +100,315 @@ header {
...
+  grid-gap: 5px;
+  margin: 5px auto;
+  width: 295px;
+  overflow: hidden;
+}
+
+.ref {
+  display: flex;
+  flex-direction: row;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] It's awesome 👍 that you have implemented a `button` tag for `See Project`. Kindly use hover and click event as per the project requirement.

File: style.css [hunk 3]
```diff
@@ -100,3 +100,315 @@ header {
...
+  margin: 16px 0 26px 0;
+  padding: 12px 12px;
+  color: #6070ff;
+  box-sizing: border-box;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: white;
+  text-transform: uppercase;
+  font-family: 'Poppins', sans-serif;
+}
```
Comment (Tufahel) [inline style.css:331]: - [ ] It's awesome 👍 that you have implemented a `button` tag for `Get My Resume`. Kindly use hover and click event as per the project requirement.

File: style.css [hunk 4]
```diff
@@ -100,3 +100,315 @@ header {
...
+  list-style: none;
+  margin-top: 20px;
+}
+
+.skill {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  overflow: hidden;
+}
```
Comment (Tufahel) [inline style.css:354]: - [ ] As per the figma design every item of this segment should be `left aligned`.  But I can see `Javascript`, `HTML` and `CSS` section is not perfectly aligned with the other component. Kindly fix this issue.

- Given Design:
- image

- Your Design:
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Joaquin123455  :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Commit message is professional.
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Joaquin123455, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights
 Awesome 👍 work done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**